pyro/distributions/transforms/affine_coupling.py

```diff
@@ -83,12 +86,16 @@ class AffineCoupling(TransformModule):
     domain = constraints.real
     codomain = constraints.real
     bijective = True
-    event_dim = 1
 
-    def __init__(self, split_dim, hypernet, log_scale_min_clip=-5., log_scale_max_clip=3.):
+    def __init__(self, split_dim, hypernet, dim=-1, log_scale_min_clip=-5., log_scale_max_clip=3.):
```

Comment:
nit: Could you separate args from kwarg with a `*,` to avoid unintentional breakage due to this change:
```py
def __init__(self, split_dim, hypernet, *, dim=-1, log_scale_min_clip=-5., log_scale_max_clip=3.):
```
This forces users to specify optional kwargs by name rather than position. This may break the 0.01 users who previously specified those args positionally, but at least it will not break their code silently.

Reply:
Done!